x-pack/packages/kbn-elastic-assistant-common/impl/schemas/evaluation/get_evaluate_route.schema.yaml

```diff
@@ -0,0 +1,40 @@
+openapi: 3.0.0
+info:
+  title: Get Evaluate API endpoint
+  version: '1'
+paths:
+  /internal/elastic_assistant/evaluate:
```

Comment:
Note: I re-used the existing `evaluate` route as that seemed the most ergonomic for fetching metadata that would be useful for executing an eval cycle via `POST /evaluate`. 

I have no plan of exposing a 'get evaluation results' endpoint at the moment (since we've been treating that data as ephemeral), so wasn't worried about any overlap here. This is an `internal` API, so there is flexibility and it can change as needed.